packages/jest-runtime/package.json

```diff
@@ -10,6 +10,7 @@
   "main": "build/index.js",
   "dependencies": {
     "@babel/core": "^7.1.0",
+    "@jest/transform": "^24.1.0",
     "babel-plugin-istanbul": "^5.1.0",
```

Comment:
I think `babel-plugin-istanbul`, `convert-source-map`, `fast-json-stable-stringify`, `micromatch`, `write-file-atomic`, and their `@types` can be removed then.

Reply:
Oops, meant to do that! Thanks